nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import errno
+import argparse
+
+def parse_args(args=None):
+    Description = 'Reformat {{ cookiecutter.name }} samplesheet file and check its contents.'
+    Epilog = """Example usage: python check_samplesheet.py <FILE_IN> <FILE_OUT>"""
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument('FILE_IN', help="Input samplesheet file.")
+    parser.add_argument('FILE_OUT', help="Output file.")
+    return parser.parse_args(args)
+
+
+def make_dir(path):
+    if not len(path) == 0:
```

Comment:
When would the `path` ever be empty?
```suggestion
    if len(path) > 0:
```

Note that it's slightly cleaner to return in these cases, to avoid having to indent the rest of the function:
```python
   if len(path) == 0:
       return
```